digitalocean-ts-loadbalanced-droplets/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Should we include a dev dependency on the node types like we do in our templates (to make it easier to work with Node APIs if someone forks and expands on this example)?

https://github.com/pulumi/templates/blob/a819c3df60524824ec3b0162dbb3ddc448fe7bfc/typescript/package.json#L3-L5

Reply:
I'm more than happy to go with your suggestion here but we need to do a sweep of all the examples if so - if you wanna open an issue about this and the version then you can assign it to me and I can look at doing it!

Reply:
OK, if we don't do it for the other examples, not a huge priority to go update all the others. Fine with keeping things as-is.